ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/event/ng/JsonRpcWebSocketAgentEventListener.java

```diff
@@ -138,6 +140,22 @@ private void initializeGitCheckoutWatcher() {
                           .sendAndSkipResult();
     }
 
+    private void initializeGitChangeWatcher() {
+        requestTransmitter.newRequest()
+                          .endpointId("ws-agent")
+                          .methodName("track:git-change")
```

Comment:
please, rename method to the "track/git-change" (this is our internal agreement)